src/views/ModelPlan/ReadOnly/Table/index.tsx

```diff
@@ -0,0 +1,295 @@
+import React, { useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  useFilters,
+  useGlobalFilter,
+  usePagination,
+  useSortBy,
+  useTable
+} from 'react-table';
+import { useQuery } from '@apollo/client';
+import { Table as UswdsTable } from '@trussworks/react-uswds';
+import { DateTime } from 'luxon';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import PageLoading from 'components/PageLoading';
+import Alert from 'components/shared/Alert';
+import GlobalClientFilter from 'components/TableFilter';
+import TablePagination from 'components/TablePagination';
+import TableResults from 'components/TableResults';
+import GetAllModelPlans from 'queries/ReadOnly/GetAllModelPlans';
+import {
+  GetAllModelPlans as GetAllModelPlansType,
+  GetAllModelPlans_modelPlanCollection as AllModelPlansType
+} from 'queries/ReadOnly/types/GetAllModelPlans';
+import globalTableFilter from 'utils/globalTableFilter';
+import {
+  translateModelCategory,
+  translateModelPlanStatus
+} from 'utils/modelPlan';
+import {
+  currentTableSortDescription,
+  getColumnSortStatus,
+  getHeaderSortIcon,
+  sortColumnValues
+} from 'utils/tableSort';
+
+const Table = () => {
+  const { t } = useTranslation('readOnlyModelPlan');
+  const { t: h } = useTranslation('modelSummary');
+  const { t: f } = useTranslation('home');
+
+  const { error, loading, data: modelPlans } = useQuery<GetAllModelPlansType>(
+    GetAllModelPlans
+  );
+
+  const data = (modelPlans?.modelPlanCollection ?? []) as AllModelPlansType[];
+
+  const columns = useMemo(() => {
+    return [
+      {
+        Header: t('allModels.tableHeading.modelName'),
+        accessor: 'modelName',
+        Cell: ({ row, value }: any) => {
+          return (
+            <UswdsReactLink to={`/models/${row.original.id}/task-list`}>
+              {value}
+            </UswdsReactLink>
+          );
+        }
+      },
+      {
+        Header: t('allModels.tableHeading.category'),
+        accessor: 'basics.modelCategory',
+        Cell: ({ value }: any) => {
```

Comment:
Just a nitpick, we generally want to try to always stay away from uses of `any`.  I know there were references from EASI that were brought over with overuse of `any`.  For this case, a more defined type would be:

```
Cell: ({ value }: { value: ModelCategory }) => {
```